### DISCUSSION TITLE
Java_org_libpag_DisplayLink_onUpdate 报错 SIGSEGV

### DISCUSSION THREAD

**Discussion by hjm1fb** [Tencent/libpag#2483] [General]
你好~ 我们的APP之前使用的是Java版的libpag SDK，后面用的是C++版本的libpag SDK. 然后经常有截图里的这个Java_org_libpag_DisplayLink_onUpdate的报错.
大部分是低RAM内存的时候出现，有时候高RAM内存时也出现。
场景是切换和预览PAG素材，调用PAG的draw方法的时候报错了。每次切换都会创建新的PAG对象实例，并销毁旧的PAG对象实例.

我看其他人都没有反馈这个问题。我猜测要么是draw方法会耗性能，设备low，内存不足导致空指针。要么是代码销毁旧pag实例时不规范，有些手机上会有内存泄漏(自测时未发现明显内存泄漏)。麻烦给一下调试和优化减少此Crash的思路建议，感谢~

Java层使用libpag SDK播放pag文件报错:
<img width="906" alt="截屏2024-09-19 11 26 30" src="https://github.com/user-attachments/assets/386b110c-d732-4b21-888d-4fc0085998f2">

C++层使用libpag.so播放pag文件报错:
<img width="1222" alt="截屏2024-09-19 11 12 31" src="https://github.com/user-attachments/assets/64693fb0-d0ac-46fa-bfd1-52f515edb26e">

**Comment by CodeJhF** [comment]
同类问题已由 #3009 继续跟踪，为避免重复排查，先关闭本条。
如果有新的复现信息，欢迎到该帖补充，或在本帖回复后重开。（stale-close）
